<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<node name="tb_autoevade_evasion_node"  pkg="tb_autoevade" type="tb_autoevade_evasion_node"    respawn="true" output="screen">
	</node>
	<node name="tb_autoevade_obstaclefinder_node"  pkg="tb_autoevade" type="tb_autoevade_obstaclefinder_node"    respawn="true" output="screen">
		<param name="update_area_sidelength"	  type="double" value="30.0"/>
		<param name="update_area_z_radius" 		 	 type="double" value="1.5" />
	</node>
	<node name="tb_autoevade_odomglobal_node"  pkg="tb_autoevade" type="tb_autoevade_odomglobal_node"    respawn="true" output="screen">
		<param name="dt_forward_horizon"	  type="double" value="2.0"/>
		<param name="dt_backward_horizon"	  type="double" value="2.0"/>
	</node>
</launch>
